[2025-03-06T03:00:03.249+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-06T03:00:03.311+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-03-05T03:00:00+00:00 [queued]>
[2025-03-06T03:00:03.323+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment scheduled__2025-03-05T03:00:00+00:00 [queued]>
[2025-03-06T03:00:03.323+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-06T03:00:03.349+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-03-05 03:00:00+00:00
[2025-03-06T03:00:03.362+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=33259) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-06T03:00:03.364+0000] {standard_task_runner.py:72} INFO - Started process 33260 to run task
[2025-03-06T03:00:03.363+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'scheduled__2025-03-05T03:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmp4nabkvmh']
[2025-03-06T03:00:03.366+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask validate_environment
[2025-03-06T03:00:03.529+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment scheduled__2025-03-05T03:00:00+00:00 [running]> on host b7f28f610775
[2025-03-06T03:00:03.796+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='datamasterylab.com' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-05T03:00:00+00:00'
[2025-03-06T03:00:03.798+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-06T03:00:03.825+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-06T03:00:03.826+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env && \n        echo "Environment is valid!"\n        ']
[2025-03-06T03:00:03.835+0000] {subprocess.py:99} INFO - Output:
[2025-03-06T03:00:03.861+0000] {subprocess.py:106} INFO - Validating environment...
[2025-03-06T03:00:03.864+0000] {subprocess.py:106} INFO - Environment is valid!
[2025-03-06T03:00:03.865+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-06T03:00:03.970+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-06T03:00:03.971+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=scheduled__2025-03-05T03:00:00+00:00, execution_date=20250305T030000, start_date=20250306T030003, end_date=20250306T030003
[2025-03-06T03:00:04.131+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-06T03:00:04.210+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-06T03:00:04.222+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
